### PR TITLE
NeLS tool assembly

### PR DESCRIPTION
**Pull request by korbinib**
This pull request includes a description of the NeLS tool assembly.
In order to display the tools in the assembly we introduce an additional tag for tools. - It might have to be discussed, wether that's desired or not.

### CODE REVIEW

**Review by wittigue — APPROVED** [review]

**Comment by bedroesb** [comment]
ok this is a nasty merge conflixt, I will try to fix it.

**Comment by bedroesb** [comment]
@korbinib can you check if i tagged the correct tools again ? :)

**Comment by bedroesb** [comment]
(during this PR we did a retagging of all the tools)

**Comment by bedroesb** [comment]
the tess ink is done automatically, and since NeLS did not give a hit in tess, it is not in the regsitry list of the tool.

File: pages/assmemblies/nels_assembly.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+The [Norwegian e-Infrastructure for Life Sciences (NeLS)](https://nels.bioinfo.no/) is an infrastructure provided by [ELIXIR Norway](https://elixir.no/). NeLS provides necessary tools for Data Management and covers [Planning](planning), [Processing](processing), [Analysing](analysing) and [Sharing](sharing) Data Life Cycle stages and offers [Data Storage](storage) capacities.
+
+## Who can use the NeLS Data Management tool assembly?
+
+NeLS and the underlying infrastructure are accessible for researchers in Norway and their collaborators. Eligible researchers can apply for storage quotas and get support through the National (Norwegian) bioinformatics support desk [contact@bioinfo.no](mailto:contact@bioinfo.no).  Most of the tools in NeLS are open source and can be reused.
+
+
+
+## For what can I use the NeLS Data Management tool assembly?
```
Comment (floradanna) [inline pages/assmemblies/nels_assembly.md:18]: Maybe you could use "you" instead of "I". I think it is suggested in the style guide.

**Review by bedroesb — APPROVED** [review]

**Pull request merged**